docs/metrics.md

```diff
@@ -0,0 +1,184 @@
+- [Collection](#collection)
+- [List of Proposed Events](#list-of-proposed-events)
+- [Adjust SDK](#adjust-sdk)
+- [References](#references)
+
+<!-- /TOC -->
+
+This is the metrics collection plan for the Lockbox android app. It documents all events that are planned to be collected through telemetry. It will be updated periodically to reflect all new and planned data collection. A similar document for the lockbox for iOS app can be found [here](https://github.com/mozilla-lockbox/lockbox-ios/blob/master/docs/metrics.md).
+
+## Analysis
+
+Data collection is done solely for the purpose of product development, improvement and maintenance.
+
+We will analyze the data described in this doc *primarily* with the purpose of (dis)confirming the following hypothesis:
+
+`If Firefox users have access to their browser-saved passwords on their mobile device but outside of the mobile browser, then they will use those passwords to log into both websites (via a browser; Firefox or otherwise) and stand-alone apps. We will know this to be true when the most frequent actions taken in the app are revealing, copying, or autofilling of credentials.`
+
+In service to validating the above hypothesis, we plan on answering these specific questions, given the data we plan to collect (see [List of Proposed Events](#list-of-proposed-events)):
+
+*Note that when we refer to retrieval of "credentials", we mean access to usernames, passwords, or both*
+
+* Are users using Lockbox to retrieve credentials?
+	* For different intervals of time (e.g. day, week, month), what is:
+		* The average rate with which a user retrieves a credential or reveals a password
+		* The distribution of above rates across all users
+* How often do users access Lockbox credentials via autofill, versus directly through the credential list provided by the app?
+* Once downloaded, do users continue to use the app? (i.e., how well are they retained?)
+	* We will count a user as retained in a given time interval if they perform one of the following actions:
+		* Display the credential list
+		* Tap a credential in the credential list
+		* Copy a credential to the clipboard
+		* Reveal a password
+		* Autofill a credential stored in Lockbox into another app
+		* Tap the URI associated with a credential (to open it in an app or browser)
+	* Since they can be performed automatically, we will **not** count a user as retained if they *only* perform the following actions (in absence of any in the list above):
+		* Unlock their credentials
+		* Sync their credentials from the Firefox desktop browser
+* Does requiring a Firefox Account constitute a roadblock to adoption?
+	* What proportion of new Lockbox users are pre-existing Firefox Account users?
+	* What proportion of users start the Account sign-in process but never complete it?
+* Does adoption of Lockbox lead to adoption of Firefox Mobile browsers (e.g. Focus)?
+	* Do users set the default browser in Lockbox to be a Firefox browser?
```

Comment:
We aren't expecting to have default browser selection in Lockbox as users can already set other browsers to be their default on Android.